File: 0455-assign-cookies/0455-assign-cookies.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        Arrays.sort(g);
        Arrays.sort(s);
        int n=g.length;
        int m=s.length;
        int e=0;
        for(int f=0;e<n && f<m;f++){
            if(g[e]<=s[f]) e++;
        }
        return e;
    }
}

File: 0455-assign-cookies/0455-assign-cookies.cs
public class Solution {
    public int FindContentChildren(int[] g, int[] s) {
        Array.Sort(g);
        Array.Sort(s);
        int n=g.Length;
        int m=s.Length;
        int e=0;
        for(int f=0;e<n && f<m;f++){
            if(g[e]<=s[f]) e++;
        }
        return e;
    }
}